docs/table_design/Data_distribution.md

```diff
@@ -27,9 +33,9 @@ Modern distributed database systems generally use four basic partitioning method
 
 To achieve more flexible data distribution, you can combine the preceding four partitioning methods based on your business requirements, such as hash-hash, range-hash, and hash-list. **StarRocks offers the following two partitioning methods:**
 
-- **Hash**: A hash-partitioned table has only one partition (the entire table is considered a partition). The partition is divided into tablets based on the bucketing column and the number of tablets that you specified.
+- **Hash**: A hash-partitioned table has only one partition (the entire table is considered a partition). The partition is divided into tablets based on the bucketing column and the number of buckets.
```

Comment:
"and the number of buckets (either manually specified or automatically configured). "??? since we have removed buckets from all examples, I think maybe it's good to add this phrase.